packages/transport/src/transports/bridge.ts

```diff
@@ -0,0 +1,341 @@
+        const response = await this._post('/read', {
+            params: session,
+        });
+
+        if (!response.success) {
+            return response;
+        }
+        const jsonData = receiveOne(this.messages, response.payload);
+
+        return this.success(jsonData);
+    }
+
+    public stop() {
+        this.stopped = true;
+        this.listening = false;
+        this.abortController.abort();
+    }
+
+    /**
+     * All bridge endpoints use POST methods
+     * For documentation, look here: https://github.com/trezor/trezord-go#api-documentation
+     */
+    private async _post(
+        endpoint: '/',
+        options: IncompleteRequestOptions,
+    ): Promise<ReturnType<(typeof bridgeApiResult)['info']>>;
+    private async _post(
+        endpoint: '/acquire',
+        options: IncompleteRequestOptions,
+    ): Promise<ReturnType<(typeof bridgeApiResult)['acquire']>>;
+    private async _post(
+        endpoint: '/call',
+        options: IncompleteRequestOptions,
+    ): Promise<ReturnType<(typeof bridgeApiResult)['call']>>;
+    private async _post(
+        endpoint: '/release',
+        options: IncompleteRequestOptions,
+    ): Promise<ReturnType<(typeof bridgeApiResult)['empty']>>;
+    private async _post(
+        endpoint: '/post',
+        options: IncompleteRequestOptions,
+    ): Promise<ReturnType<(typeof bridgeApiResult)['empty']>>;
+    private async _post(
+        endpoint: '/read',
+        options: IncompleteRequestOptions,
+    ): Promise<ReturnType<(typeof bridgeApiResult)['call']>>;
+    private async _post(
+        endpoint: '/listen',
+        options?: IncompleteRequestOptions,
+    ): Promise<ReturnType<(typeof bridgeApiResult)['devices']>>;
+    private async _post(
+        endpoint: '/enumerate',
+        options: IncompleteRequestOptions,
+    ): Promise<ReturnType<(typeof bridgeApiResult)['devices']>>;
+    private async _post(endpoint: BridgeEndpoint, options: IncompleteRequestOptions): Promise<any> {
+        const { abortable, ...restOptions } = options;
+
+        const requestTimeout = setTimeout(() => {
+            this.abortController.abort('request timeout');
+            /**
+             * All requests (except /listen endpoint) to local instance of bridge
+             * don't have right to take long. If so, this is probably some kind of
+             * error with bridge. I have seen this to happen that the first request
+             * to bridge ('/' endpoint) did not respond. This leads to not
+             * emitting transport.start event in time and suite rendering
+             * "loading takes too long" error screen.
+             */
+        }, 5000);
+
+        const response = await bridgeApiCall({
+            ...restOptions,
+            signal: abortable ? this.abortController.signal : undefined,
+            method: 'POST',
+            url: `${this.url + endpoint}${options?.params ? `/${options.params}` : ''}`,
```

Comment:
use fetch `timeout` option instead of `requestTimeout`

Reply:
b18599a494870c84b7a6733fc8505b343bbd8347 but lets keep this open, I need to test if it really works